workflow/snakemake_rules/upload.smk

```diff
@@ -57,24 +35,34 @@ def compute_files_to_upload(wildcards):
 
         files_to_upload["biosample.tsv.zst"] =           f"data/{database}/biosample.tsv"
         files_to_upload["duplicate_biosample.txt.zst"] = f"data/{database}/duplicate_biosample.txt"
+
     elif database=="gisaid":
         files_to_upload["additional_info.tsv.gz"] =     f"data/{database}/additional_info.tsv"
         files_to_upload["flagged_metadata.txt.gz"] =    f"data/{database}/flagged_metadata.txt"
 
         files_to_upload["additional_info.tsv.zst"] =     f"data/{database}/additional_info.tsv"
         files_to_upload["flagged_metadata.txt.zst"] =    f"data/{database}/flagged_metadata.txt"
-
+        
+    # Include upload of raw NDJSON if we are fetching new sequences from database
+    if config.get("fetch_from_database", False):
+        files_to_upload.update({
+            f"{database}.ndjson.xz": f"data/{database}.ndjson",
+            f"{database}.ndjson.zst": f"data/{database}.ndjson",
+        })
+        if database=="genbank":
+            files_to_upload.update({
+                "biosample.ndjson.gz": f"data/biosample.ndjson",
+                "biosample.ndjson.zst": f"data/biosample.ndjson",
+            })
     return files_to_upload
 
-def files_to_upload(wildcards):
-    files_to_upload_dict = compute_files_to_upload(wildcards)
-    return [f"data/{database}/{remote_file}.upload" for remote_file in files_to_upload_dict.keys()]
+files_to_upload = compute_files_to_upload()
 
 
 rule upload_single:
-    input: lambda wildcards: compute_files_to_upload(wildcards)[wildcards.remote_filename]
+    input: lambda w: files_to_upload[w.remote_filename]
     output:
-        "data/{database}/{remote_filename}.upload"
+        "data/{remote_filename}.upload"
```

Comment:
Removing the `{database}/` prefix here will break the triggers for the ncov builds 
https://github.com/nextstrain/ncov-ingest/blob/eafb1d5d32b3d613a6c958815b61625ab9dd88e6/workflow/snakemake_rules/trigger.smk#L17-L18

Is there any particular reason to remove the database prefix here? I think it's better organization to keep the database files in separate directories.

Reply:
I see - I didn't know these `.upload` touch files were used and the extra `{database}` seemed misleading when not all files were actually in the `{database}` folder (data/biosample.ndjson for example isn't under data/genbank/biosample.ndjson).

I'll happily change!